wiki/Game_mode/osu!/id.md

```diff
@@ -0,0 +1,285 @@
+
+# osu! (mode permainan)
+
+*Untuk kegunaan lain, lihat [osu! (disambiguasi)](/wiki/Disambiguasi/osu!).*
+
+![Tampilan antarmuka permainan osu!](/wiki/shared/osu-gameplay.jpg "Tampilan antarmuka permainan osu!")
+
+Mode permainan pertama yang namanya terinspirasi dan didasari dari seri ouendan.
+
+## Apa itu "Ouendan"?
+
+*[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* merupakan sebuah permainan pada konsol *Nintendo DS* yang mendasari dan menginspirasi awal terbentuknya osu!. Permainan tersebut pada intinya hanya didasari pada tiga aturan yang harus kamu ikuti di dalamnya: sentuh circle-circle yang muncul pada layar, ikuti slider-slider sesuai dengan alur yang tergambar, dan putar spinner yang ada secepat mungkin. Elemen-elemen yang ada di dalam permainan tersebut akan secara silih berganti muncul di dalam layar permainan sejalan dengan irama yang hadir dari lagu yang sedang dimainkan. Permainan tersebut tampak seperti ini pada layar *DS*:
+
+![Contoh tampilan permainan Osu! Tatakae! ouendan pada layar Nintendo DS](/wiki/shared/Ouendan.jpg "Contoh tampilan permainan Osu! Tatakae! ouendan pada layar Nintendo DS")
+
+Terdapat dua layar *DS* yang aktif dalam satu waktu, di mana layar *DS* bagian atas menampilkan sebuah kisah bergambar dan layar DS bagian bawah menampilkan keseluruhan circle, slider, dan spinner untuk kamu mainkan. Setiap level pada ouendan memiliki ceritanya tersendiri yang pada umumnya mengisahkan seseorang yang berada dalam kesulitan. Di tengah badai permasalahan inilah pasukan Ouendan hadir menolong. Melalui kekuatan ajaib yang mereka miliki, kamu harus menolong orang-orang tersebut untuk dapat keluar dari permasalahan yang mereka hadapi.
+
+## ![](/wiki/shared/mode/osu.png) Gameplay
+
+*Tutorial: [new beginnings (Dibundel ke dalam instalasi osu!)](https://osu.ppy.sh/beatmapsets/1011011).*
+
+Dalam bermain osu!, secara umum kamu dituntut untuk:
+
+- Dapat beradaptasi dan bereaksi terhadap stimulus-stimulus visual.
+- Dapat memahami ketukan-ketukan dan irama-irama pada lagu.
+- Dapat mengontrol peranti input yang digunakan.
+- Dapat menakar timing objek-objek yang muncul di layar.
+- Dapat melakukan gerakan rotasi yang berkelanjutan pada spinner.
+- Dapat senantiasa menjaga fokus dan konsentrasi mental (terutama untuk lagu-lagu berdurasi panjang, seperti maraton).
+- Dapat melakukan koordinasi yang baik antara penglihatan mata dan pergerakan kedua ruas tangan.
+
+Hal-hal diatas merupakan segelintir aspek penting yang harus dimiliki oleh seseorang untuk menjadi *pemain osu! profesional*.
+
+Sebelum melangkah lebih jauh, ada baiknya bagi kamu untuk terlebih dahulu mempelajari dasar-dasar teori seputar objek-objek permainan yang ada di osu! sebagai berikut.
+
+### Pemilihan Lagu
+
+Untuk memilih mode permainan yang ada di osu!, tekan kombinasi tombol `Ctrl`+`1`.
+
+Alternatif lainnya, kamu dapat menekan tombol `Mode` dan pilih `osu!`.
+
+### Dasar-dasar Permainan
```

Comment:
```suggestion
### Dasar-dasar Gameplay
```

Reply:
since line 19 using Gameplay as the main phase, should be consistent

Reply:
I use "Permainan" because i follow this article https://osu.ppy.sh/wiki/id/FAQ

Reply:
then line 19 should be `Permainan` 🤔

Reply:
yes, i have changed it